src/components/SearchDrawer.tsx

```diff
@@ -46,13 +64,13 @@ const SearchDrawer = (): JSX.Element | null => {
                             id: 'layout.organize.search.placeholder',
                         }) }
                         value={ currentText }
-                        width="100%"
+                        variant="outlined"
                     />
-                    <View isHidden={ !drawerOpen }>
-                        <Text>
+                    <Box display={ drawerOpen ? 'block' : 'none' }>
```

Comment:
Is there any MUI component we could use instead of the custom solution for the `SearchDrawer`?